playground/src/main/java/org/navgurukul/playground/ui/PythonPlaygroundActivity.kt

```diff
@@ -368,6 +382,200 @@ class PythonPlaygroundActivity : AppCompatActivity() {
 
     }
 
+    private fun createErrorUI() {
+        try {
```

Comment:
no reason to have this under try catch block

Reply:
removed